README.md

```diff
@@ -779,7 +779,8 @@ Miscellaneous of tools that provide some funny/aesthetical functionality (animat
 * [Draw](https://github.com/maaslalani/draw) - draw is an simple drawing tool in the terminal. Hold your mouse down and move it across the screen to draw anything you want!
 * [fortune](http://software.clapper.org/fortune/) - Generates random messages fetched from a quotation database.
 * [HARRY_POTTER_ALIASES](https://github.com/P-Y-R-O-B-O-T/HARRY_POTTER_ALIASES) - Harry Potter-themed aliases a magical journey through the wizarding world of terminal commands.
-* [hollywood](https://github.com/dustinkirkland/hollywood) - Runs a script turning your Linux terminal into a Hollywood style real-time hacking terminal.
+* [hollywood](https://github.com/dustinkirkland/hollywood) - Runs a script turning your Linux terminal into a Hollywood style real-time hacking terminal. 
+* [Bollywood](https://github.com/abloch/bollywood)) - Runs terninal screencasts in multiple panes, resulting in another real-time Hollywood-style real-time hacking terminal.
```

Comment:
```suggestion
* [Bollywood](https://github.com/abloch/bollywood) - Runs terninal screencasts in multiple panes, resulting in another real-time Hollywood-style real-time hacking terminal.
```

Reply:
I am not sure what this is. For sure it is not a patch that is done on the requested CSV file...

Reply:
I simply reported an issue I found in the README, but yes if the README is generated from the CSV file in data folder then. There is a problem.